### DISCUSSION TITLE
Assemblies in sub folder not referred

### DISCUSSION THREAD

**Discussion by PramBellikoth** [NuGet/Home#12805] [Q&A]
I am trying to create a nuget package where the assemblies are added in to subfolders as below

>      lib
>        -net48
>            -Subfolder1
>                - Assembly1.dll
>             -Subfolder2
>                 -Assembly2.dll
>              CommonAssembly1.dll
> 

     When I add this as package reference to sample application, only the CommonAssembly1.dll is resolved and the assemblies in subfolders are ignored.
      Is this expected behavior or is there any way to make nuget resolve the assemblies in the subfolders?

  thank you

**Comment by bilbothebaggins** [comment]
As far as I can tell, this is expected behavior.
That is, only stuff in `lib/TFM/*.*` will be included, *not* files in `lib/TFM/SubDir/*.*`. (TFM: https://learn.microsoft.com/en-us/nuget/reference/target-frameworks)

I don't know where to find the official docs for this though.
